resources/responses.js

```diff
@@ -444,9 +444,9 @@ let Responses = {
   getBehind: (sev) => {
     let obj = {};
     obj[defaultAlertText(sev)] = {
-      en: 'Get Behind',
-      de: 'Hinter ihn',
-      fr: 'Passez derrière',
+      en: 'Get Behind Boss',
```

Comment:
Subjectively, I think "boss" is implied here and on many other triggers, and I prefer the original.  I think if there's something else to "get behind" (or look towards, etc, etc) then that can be specified differently.

Reply:
ok i correct